api/features/versioning/models.py

```diff
@@ -128,3 +129,12 @@ def publish(
         if persist:
             self.save()
             environment_feature_version_published.send(self.__class__, instance=self)
+
+    def clone(self, environment: "Environment") -> "EnvironmentFeatureVersion":
```

Comment:
I'm aware that the naming's already pretty deeply integrated but I really think `clone_to_environment` or `migrate_to_environment` would be better.

Reply:
Sure, fair point. On it.

Reply:
Done.